src/main/java/steps/TestResultsSteps.java

```diff
@@ -698,8 +698,8 @@ public void validateCertificateIsGenerated(String uuid, String vin) {
     }
 
     @Step
-    public void insertRecordInDynamo(String json, String table) {
-        AwsUtil.insertJsonInTable(json, table);
+    public void insertRecordInDynamo(String json, String table, String primaryKey) {
```

Comment:
why did you replace instead of overloading this method ?!

Reply:
Because I done see the purpose in having the method without the primary key parameter since in the method insertJsonInTable we need the primary key parameter in order to make it work for insertions in any table.